recorder/build.gradle

```diff
@@ -47,6 +53,14 @@ android {
     buildTypes {
         debug {
             applicationIdSuffix ".debug"
+
```

Comment:
Would you like to extract this block to smth like `configureFirebase("debug")`/`configureFirebase("release")`?

Reply:
No, I wouldn't.

Reply:
![image](https://user-images.githubusercontent.com/2265924/76764886-a34a2980-67a6-11ea-831a-dc6a6171cd54.png)

Reply:
👍